config.yaml

```diff
@@ -17,21 +27,16 @@ common:
    --plugin-option=tmpfs:keep_mounted=True
    --plugin-option=tmpfs:max_fs_size=32g
    --plugin-option=tmpfs:required_ram_mb=39800"
- commit_updates: True
- push_updates: True
-build_packages:
- packages:
- packages_repos_target_path: "/var/lib/host-os/repositories"
- keep_build_dir: False
- result_dir:  "./result"
- force_rebuild: False
 build_release_notes:
  repo_url: "https://github.com/open-power-host-os/open-power-host-os.github.io.git"
  repo_branch: "master"
+ push_repo_url: "git@github.com:open-power-host-os/open-power-host-os.github.io.git"
```

Comment:
Thanks, this is clearer to me now. I thought about using the URL of our bot user git repo as default here. However, as I discussed with Olav today, running update-versions and build-release-versions subcommands won't work out of the box anyway because the user must have the private SSH key of the GitHub user and a SSH config setup in the system running host_os.py. IMHO either command line should parametrize this or at least we should document this so that these commands work in the command line and we do not have a dependency with Jenkins here, but Olav told me this has low priority. I'll just leave the attribute blank for now.